### PR TITLE
CLC-5228 - Web Accessibility: Apply Accessibility Guidelines to 'Official Sections' Tool

### PR DESCRIPTION
**Pull request by redconfetti**
https://jira.ets.berkeley.edu/jira/browse/CLC-5228

### CODE REVIEW

**Comment by raydavis** [comment]
Another educational PR. Thanks, @redconfetti !

File: src/assets/templates/canvas_embedded/course_manage_official_sections.html [hunk 1]
```diff
@@ -8,16 +8,18 @@
...
 
-    <h2 class="cc-page-site-provisioning-header cc-page-site-provisioning-header2">Official Sections</h2>
+    <h1 class="cc-page-course-official-sections-header1">Official Sections</h1>
 
     <!-- ############################################################################################# -->
 
     <div data-ng-if="currentWorkflowStep === 'preview'">
 
-      <div data-ng-show="jobStatusMessage !== ''" class="cc-notice-box cc-radius" data-ng-class="{'cc-notice-error':(lastJobStatus !== 'sectionEditsCompleted'), 'cc-notice-success':(lastJobStatus === 'sectionEditsCompleted')}">
+      <div data-ng-show="jobStatusMessage !== ''" class="cc-notice-box cc-radius" data-ng-class="{'cc-notice-error':(lastJobStatus !== 'sectionEditsCompleted'), 'cc-notice-success':(lastJobStatus === 'sectionEditsCompleted')}" id="cc-page-course-official-sections-job-status-notice" aria-hidden="{{jobStatusMessage === ''}}">
```
Comment (christianv) [inline src/assets/templates/canvas_embedded/course_manage_official_sections.html]: @redconfetti would it be possible to use `ng-attr` for dynamic properties?
e.g. `ng-attr-aria-hidden`

Definitely test this out before committing.

Comment (redconfetti) [inline src/assets/templates/canvas_embedded/course_manage_official_sections.html]: @christianv I tested it, it didn't work with `data-ng-attr-aria-hidden` or `ng-attr-aria-hidden`

Comment (christianv) [inline src/assets/templates/canvas_embedded/course_manage_official_sections.html]: @redconfetti that's very interesting. Did you see an error in the console or it didn't to anything at all?

We do use `ng-attr` pretty successfully in other places:
https://github.com/ets-berkeley-edu/calcentral/blob/172d183e35e90ad49426fda3ff8548fbd1948796/src/assets/templates/cars_activity.html#L128

Comment (redconfetti) [inline src/assets/templates/canvas_embedded/course_manage_official_sections.html]: @christianv No, there is no error in the console. But I don't see the `aria-hidden="false"` attribute added to the element in the DOM.

```html
<div data-ng-show="jobStatusMessage !== ''" class="cc-notice-box cc-radius ng-hide cc-notice-error" data-ng-class="{'cc-notice-error':(lastJobStatus !== 'sectionEditsCompleted'), 'cc-notice-success':(lastJobStatus === 'sectionEditsCompleted')}" id="cc-page-course-official-sections-job-status-notice" ng-attr-aria-hidden="jobStatusMessage === ''">
...
</div>
```

Comment (christianv) [inline src/assets/templates/canvas_embedded/course_manage_official_sections.html]: @redconfetti did you also try: `data-ng-attr-aria-hidden="{{jobStatusMessage === ''}}"` ?

Comment (redconfetti) [inline src/assets/templates/canvas_embedded/course_manage_official_sections.html]: @christianv Yes, it did. So I have to interpolate the value, it's not natively going to evaluate the contents. That's odd, I didn't see it even add the 'aria-hidden' attribute, even with the actual literal expression, before. It's working now. I've made PR #3760 to resolve this.

**Pull request merged**